ui/helpers/utils/settings-search.js

```diff
@@ -350,6 +350,13 @@ export function getSettingsRoutes(t) {
         route: `${EXPERIMENTAL_ROUTE}#token-description`,
         icon: 'fa fa-flask',
       },
+      {
```

Comment:
I think this should just be added as part of the initial definition of the settingRouteList array. This if else block should probably just be used for these two token detection related settings.

Reply:
Good catch, fixed